pkg/metrics/percentile/gk_array.go

```diff
@@ -194,6 +194,84 @@ func (s GKArray) interpolatedQuantile(q float64) float64 {
 	return weightBelow*s.Entries[indexBelow].V + weightAbove*s.Entries[indexAbove].V
 }
 
+// Quantiles accepts a slice of quantiles and returns the epsilon estimates of
+// elements at those quantiles.
+func (s GKArray) Quantiles(qValues []float64) []float64 {
+	quantiles := make([]float64, 0, len(qValues))
+	if s.Count == 0 {
+		for _ = range qValues {
+			quantiles = append(quantiles, math.NaN())
```

Comment:
Would returning 0 be more correct (and simpler)?

e.g. return `make([]float64, len(qValues))`

Reply:
I don't think so, since 0 is meaningful. E.g., if the values were [0, 0, 0, ...] the the quantiles should return 0, but if there were no values quantiles are meaningless.